### ISSUE TITLE
Password Input field turns white on setting type="password" in <FormField> attribute

### ISSUE THREAD

**Issue by nefario7** [grommet/grommet#4276]
I am creating a sign-in page, which has a simple email and password input field. I have used <FormField> to create both the input boxes, and have formatted them to a custom theme. But, when I try to set the 'type' attribute for both the fields, the formatting of the fields changes completely and the field background changes to white, because of which whatever I type in the field is not visible as the text input colour is also white.

### Expected Behavior
This is how the fields look normally when I don' use the 'type' attribute:
![image](https://user-images.githubusercontent.com/64963908/86919621-27556200-c146-11ea-8bf4-6a67afa11c67.png)

But, because of not using the 'type' attribute, I don't get the password in dots. It just displays the password like a normal text field:
![image](https://user-images.githubusercontent.com/64963908/86920092-d134ee80-c146-11ea-8ca2-81b3a006fc7c.png)

### Actual Behavior
This is how the fields looked when I use the 'type' attribute: (You can see that the typed email is not visible, but when I highlight it by selecting it, you can see that it's there)
![image](https://user-images.githubusercontent.com/64963908/86919807-697ea380-c146-11ea-978a-49115da4e944.png)

### Code Screenshot
![image](https://user-images.githubusercontent.com/64963908/86920539-91223b80-c147-11ea-9554-359febec4624.png)

And I just add - type="text" or type="password" as attributes.

### Your Environment

- Grommet version: grommet ^2.13.0
- Browser Name and version: Microsoft Edge Chromium Version 83.0.478.58 (Official build) (64-bit)
- Operating System and version (desktop or mobile): Windows 10 Home Version 2004

**Comment by nefario7** [comment]
I have also tried this with <TextInput> component, and I get the same results with that. If someone can help with how I should be creating a password field, that'll be really helpful.

**Comment by skad0** [comment]
Hi, FormField doesn't have property `type` as you can see here: https://v2.grommet.io/formfield
I think there is also an example of usage. 
e.g.
```jsx
<FormField label="Field label">
  <TextInput placeholder="type here" type="password" />
</FormField>
```

**Comment by nefario7** [comment]
@skad0 As I already mentioned in a comment earlier, I have also tried it with the `<TextInput>` component as you suggested. Here are the results of using this code :
```javascript
<FormField value={password} onChange={(event) => onChangeHandler(event)} required>
    <TextInput plain name="userPassword" placeholder="Password" type="password"/>
</FormField>
```

![image](https://user-images.githubusercontent.com/64963908/87024643-06007e80-c1f7-11ea-9095-4e187e52c5b1.png)
(Here the password is again white, and is only visible when I highlight it by selecting it)

**Comment by skad0** [comment]
Sorry, didn't understand you first. 
Now I see what is the issue and I don't have enough info.
You can try to understand which style make it white with dev tools, maybe there are missing color definition in your custom theme or missed color in grommet default theme.

**Comment by IanKBovard** [comment]
This is most likely due to an issue with theme. Could you give me an example in codesandbox. You can use this.

https://codesandbox.io/s/modest-tdd-jhnwo?file=/src/App.js

**Comment by nefario7** [comment]
@IanKBovard So, I tried implementing it in the sandbox and there it seems to be working fine. I don't know what the issue is with my codebase, I removed the custom theme I applied and even then issue was persistent. I guess the issue must be with some styling in one of the CSS files. But, anyway thanks a lot for the help!

As the issue is probably not because of anything related to grommet, I'll close this issue for now.
